examples/common/pigweed/RpcService.cpp

```diff
@@ -0,0 +1,118 @@
+/*
+ *
+ *    Copyright (c) 2020 Project CHIP Authors
```

Comment:
2021

here and in the other files that were added.

Reply:
done